docs/configuration.rst

```diff
@@ -2339,23 +2339,40 @@ Default value: True
 
 Can be overwritten for each single need by setting :ref:`need_collapse`.
 
-.. _needs_per_id_build_path:
+.. _needs_build_json_per_id_path:
 
-needs_per_id_build_path 
-~~~~~~~~~~~~~~~~
+needs_build_json_per_id_path 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 This option sets the location of the set of ``needs.json`` for every needs-id.
 
 Default value: needs_id
 
+Example:
 
-.. _needs_per_id:
+.. code-block:: python
 
-needs_per_id 
-~~~~~~~~~~~~~~~~
+    needs_build_json_per_id_path = "needs_id"
 
-Exports forder ``needs_per_id_build_path`` include all file ``needs.json`` for every needs-id.
+.. hint::
+
+   The created ``needs_id`` folder gets stored in the ``outdir`` of the current builder. The final location is e.g. ``_build/needs_id``
 
-This option works like needs_build_json. 
+.. _needs_build_json_per_id:
+
+needs_build_json_per_id 
```

Comment:
Please document first `needs_build_json_per_id` then `needs_build_json_per_id_path`, as `needs_build_json_per_id_path` is based on `needs_build_json_per_id`